tests/testthat/test-fetch.R

```diff
@@ -0,0 +1,51 @@
+test_that('Test fetchMetadata', {
```

Comment:
Do we need to preface every function name with "Test "? I don't think our other packages do that.